guides/hack/00-overview/03-async.md

```diff
@@ -1,85 +1,15 @@
 ### Async Programming in Hack

```

Comment:
I made a comment about this elsewhere, but we have a problem currently where we're autogenerating an <h2> page title for all guides, and then below that we have the <h1> title from the Markdown - we need to figure out whether we ditch the autogen title completely, or downgrade the manual titles to <h2> and upgrade the autogen to <h1>